### PR TITLE
Fix #16 - histograms of activations and gradients

### LINKED ISSUE #16: Instrument histograms of activations and gradients

### PR DESCRIPTION
**Pull request by tilmankamp**
@kdavis-mozilla : Please review

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
Not finished with the review but here are the first few comments.

I can't do inline comments so I'll add them here...

- First cell has been is converted to a code cell and not markdown cell
- `dropout_rate` was set to 0.00 and not 0.05
- I'd guess you compute gradient twice. Use `compute_gradients` and `apply_gradients` and not `minimize`. See [[1]](https://www.tensorflow.org/versions/r0.10/api_docs/python/train.html#processing-gradients-before-applying-them)

**Comment by kdavis-mozilla** [comment]
LGTM, but we need to merge our pull requests. 

I'd guess mine should go first, just because it's a bigger change, then integrate your changes.

**Comment by kdavis-mozilla** [comment]
I guess as you put this into the other pull request we can close this?

**Comment by kdavis-mozilla** [comment]
@tilmankamp I'm just going to remove this pull request as it's already part of master from the issue23 pull request.

**Comment by kdavis-mozilla** [comment]
I'm going to delete the branch as the changes are now in master.
